### DISCUSSION TITLE
请问会员版支持 GPT 模型的翻译吗？具体支持哪些模型？是否可以选择呢？

### DISCUSSION THREAD

**Discussion by notthere-2023** [immersive-translate/immersive-translate#2198] [Help]
### 概述 | Summary

请问会员版支持 GPT 模型的翻译吗？具体支持哪些模型？是否可以选择呢？

**Comment by keyboard3** [comment]
价格页下拉可查。当前不可选
![QQ_1729346998893](https://github.com/user-attachments/assets/9f24eb59-b656-4652-a41b-9c135b47c3be)

**Reply by notthere-2023** [reply]
谢谢！

**Comment by notthere-2023** [comment]
![2230860bfd9f954d4deab40fc1b06a0](https://github.com/user-attachments/assets/118c8ccd-c2f9-44eb-bcf3-86402c93edff)

目前只看到这些，没看到 4o 或者 4 等模型

**Comment by keyboard3** [comment]
你这是AI提示词专家。pro模型是无法选择的

**Reply by notthere-2023** [reply]
谢谢你
